---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    entando.org/entando-resource-namespace: "my-namespace"
    entando.org/entando-resource-kind: "EntandoApp"
    EntandoApp: "my-app"
  name: "my-app-37842641"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "entando.org/v1"
    kind: "EntandoApp"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "15e0cffd-dd4d-46c0-a1cd-e30d963fb3ee"
action: "FAILED"
count: 1
firstTimestamp: "2021-07-30T14:36:13Z"
involvedObject:
  apiVersion: "entando.org/v1"
  kind: "EntandoApp"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "5886500"
  uid: "15e0cffd-dd4d-46c0-a1cd-e30d963fb3ee"
lastTimestamp: "2021-07-30T14:36:13Z"
message: "The deployment of EntandoApp my-namespace/my-app failed due to Fake Controller\
  \ Exception. Fix the root cause and then trigger a redeployment by adding the annotation\
  \ 'entando.org/processing-instruction: force'"
reason: "Failed"
related:
  apiVersion: "v1"
  kind: "Pod"
  name: "entando-k8s-entando-k8s-app-con-m5mgq-51-from-build-pack--4tkvh"
  namespace: "controller-namespace"
source:
  component: "entando-k8s-app-controller"
type: "Error"
